### ISSUE TITLE
nbconvert failure on jupyterhub 

### ISSUE THREAD

**Issue by epifanio** [ipython-contrib/IPython-notebook-extensions#408]
I just upgraded to the latest version (git master) of python, jupyter, notebook, jupyterhub, nbconvert. And I installed IPython-notebook-extensions appliend the changes from: 
  
https://github.com/ipython-contrib/IPython-notebook-extensions/pull/402

When tring to save as html, or pdf i have this log :
```
Could not construct Exporter: The 'parent' trait of a CodeFoldingPreprocessor instance must be a Configurable, but a value of class 'nbconvert.exporters.pdf.PDFExporter' (i.e. <nbconvert.exporters.pdf.PDFExporter object at 0x7fd7f39f8908>) was specified.
```
and to html:
```
Could not construct Exporter: The 'parent' trait of a CodeFoldingPreprocessor instance must be a Configurable, but a value of class 'nbconvert.exporters.html.HTMLExporter' (i.e. <nbconvert.exporters.html.HTMLExporter object at 0x7fd7f0a17f60>) was specified.
```

While this is the full log if I try to use nbcovert from the command line:
https://gist.github.com/50e5e8568c08b4f245cb

I guess it is failing because of the changes in the config files which in my case are: 
 [```jupyter_nbconvert_config.py``` ](https://gist.github.com/epifanio/1238d7255627b3c0564a) and [```jupyter_nbconvert_config.json``` ](https://gist.github.com/afca6d4a3ae15bf3c962)

in the log i see: ```/home/epinux/.local/share/jupyter/templates``` but this file is no more on my system and also: [jupyter_nbconvert_config-json-L8](https://gist.github.com/epifanio/afca6d4a3ae15bf3c962#file-jupyter_nbconvert_config-json-L8) i think ```/user/``` is a typo and should be ```/usr/```.

**Comment by juhasch** [comment]
The error seems to be due to recent changes in places other than nbconvert. I don't see this with the latest master of nbconvert.

The templates path is set using the juyper_data_dir as base director:
```Python
from jupyter_core.paths import jupyter_data_dir
jupyter_data_dir()
```
Can you check what this call returns for you ?

**Comment by epifanio** [comment]
starting a notebook as per user ```epinux``` it returns:

```
'/home/epinux/.local/share/jupyter'
```

if I log out and log in with an other user  say ```user2``` it is giving me:

```
'/home/user2/.local/share/jupyter'
```

I guess the templatepath for the jupyterhub (multiuser) should point to the global directory (which is available for all the users) that's:

```
/usr/loacl/share/jupyter/
```

**Comment by juhasch** [comment]
Yes it makes sense to check, if the path exists and use another one instead.
Can you check what this returns:
```Python
import jupyter_core.paths
jupyter_core.paths.jupyter_config_path()
```

**Comment by epifanio** [comment]
Running:
```
import jupyter_core.paths
jupyter_core.paths.jupyter_config_path()
```
From a py2 notebook, as user "epinux" it returns:
```
['/home/epinux/.jupyter',
 '/home/epilib/Envs/env1/etc/jupyter',
 '/usr/local/etc/jupyter',
 '/etc/jupyter']
```

**Comment by juhasch** [comment]
Likely solved by #492. If not, please reopen this issue.
